src/app/components/DeleteModal/DeleteModal.test.tsx

```diff
@@ -110,6 +110,26 @@ describe("Delete Modal component", () => {
     expect(onConfirm).toBeCalledTimes(1);
   });
 
+  test("should error when deletion was not confirmed properly", async () => {
+    const onConfirm = jest.fn();
+    const onCancel = jest.fn();
+
+    const { comp } = setupModal({
+      showDialog: true,
+      onCancel,
+      onConfirm,
+    });
+    await waitForI18n(comp);
+
+    expect(await comp.findByText(title)).toBeInTheDocument();
+    expect(comp.getByText("Delete")).toBeDisabled();
+
+    fireEvent.change(comp.getByLabelText(`Type ${testName} to confirm.`), {
+      target: { value: "WRONG_VALUE" },
+    });
+    expect(comp.getByText("Delete")).toBeDisabled();
```

Comment:
What do you think about an addition of an assertion that the "Cancel" button is enabled?

Reply:
sure, good idea, pushed in a separate commit, thank you